web/gallery/tool/run_instrumentation_test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
nit: Can probably lose this file as well, since it appears to relate to Android specifically.